packages/kuma-gui/src/app/application/services/storage/index.ts

```diff
@@ -0,0 +1,21 @@
+export default (prefix: string = 'kumahq-app', storage: Storage = window.localStorage) => {
+  const get = async (key: string): Promise<object> => {
+    try {
+      return JSON.parse(storage.getItem(`${prefix}:${key}`) ?? '{}')
+    } catch (e) {
+      console.error(e)
+    }
+    return {}
+  }
+  const set = async (key: string, value: object): Promise<object> => {
+    try {
+      storage.setItem(`${prefix}:${key}`, JSON.stringify(value))
+      return value
+    } catch (e) {
+      console.error(e)
+    }
+    return {}
+  }
+
+  return { get, set }
+}
```

Comment:
If we want to get rid of `console.error`s entirely (and throw/catch errors or something else), we'd have to decide what to do here. Thinking about the type of error we'd get here, I'd be almost 100% on just lint-ignoring these ones.